juriscraper/pacer/free_documents.py

```diff
@@ -103,26 +99,25 @@ def download_pdf(self, pacer_case_id, pacer_document_number):
 
         Note that this doesn't support attachments yet.
         """
+        timeout = (60, 300)
         url = make_doc1_url(self.court_id, pacer_document_number, True)
         data = {
             'caseid': pacer_case_id,
             'got_receipt': '1',
         }
+
         logger.info("GETting PDF at URL: %s with params: %s" % (url, data))
-        r = self.session.get(
-            url,
-            params=data,
-            headers={'User-Agent': 'Juriscraper'},
-            verify=False,
-            timeout=300,
-        )
+        r = self.session.get(url, params=data, timeout=timeout)
```

Comment:
Haven't figured out if it does, and the change to the tests might make it moot, but in looking through the docs for `requests` you can send the timeout a tuple of a connection timeout and a response timeout. That way we don't burn 5 mins waiting to connect potentially even before a download can start. I noticed in some failures in Python3.x it was raising ConnectionErrors which seemed like we shouldn't wait nearly as long for connections as for downloads. No idea if it helps yet.